topology/mask.h

```diff
@@ -339,13 +340,35 @@ class EllipseMask : public Mask< D >
     , polar_axis_( polar_axis )
     , azimuth_angle_( azimuth_angle )
     , polar_angle_( polar_angle )
+    , pi_( 3.141592653589793 )
```

Comment:
It is better to use `numerics::pi` from `numerics.h` for consistency across the code.